cpp/include/cudf/reduction.hpp

```diff
@@ -75,15 +64,16 @@ std::unique_ptr<scalar> reduce(
  * @throws `cudf::logic_error` if column datatype is not numeric type.
  *
  * @param[in] input The input column view for the scan
- * @param[in] op The operation of the scan
+ * @param[in] agg unique_ptr of the aggregation operator applied by the scan
  * @param[in] inclusive The flag for applying an inclusive scan if true,
  *            an exclusive scan if false.
  * @params[in] mr The resource to use for all allocations
```

Comment:
Missing `skipna` @param[in] documentation. Also agree that it would be good to eliminate `bool` parameters, as @codereport mentions. And I think `skipna` could have a clearer name. Is it skipping nulls? Then `skip_nulls` would be better.

Reply:
renamed to `skip_nulls` and also converted to enum class.